nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -89,6 +89,11 @@ services.mysql.initialScript = pkgs.writeText "mariadb-init.sql" ''
       When MariaDB data directory is just upgraded (not initialized), the users are not created or modified.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     <varname>services.postfix.sslCACert</varname> was replaced by <varname>services.postfix.tlsTrustedAuthorities</varname> which now defaults to system certifcate authorities.
```

Comment:
This should be moved to the section "Backward Incompatibilities".

Reply:
It would also be nice to mention the fact that even if `smtpd_tls_CAFile` is no longer configured, there is no impact since this configuration was not taken into account because `smtpd_tls_ask_ccert` defaults to `no`.

Reply:
It would be also good to mention what the implication for the user for this change is.

Reply:
@nlewo moved section, added
> services.postfix.tlsTrustedAuthorities won't configure smtpd_tls_CAFile by default anymore. This should not have any impact since smtpd_tls_CAFile is only relevant for servers with smtpd_tls_ask_ccert = yes which needs to enabled explicitly by config.

@Mic92 added
> This enables postfix to validate TLS certificates for outgoing smtp connections by default. This should not have any implications since most servers are running in opportunistic TLS mode which allows untrusted TLS connections to smtp servers.
For more details information to enforce only trusted TLS connections see [smtp_tls_security_level](http://www.postfix.org/postconf.5.html#smtp_tls_security_level).